File: VB.NET/windows/label/label/Form1.vb
Public Class Form1
    Dim a, b, c As Integer
    Dim s1 As String = TextBox1.Text
    Dim s2 As String = TextBox2.Text

   

   

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        a = CInt(s1)
        b = CInt(s2)
        c = a + b
        MsgBox("Addition=" + c.ToString)
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        a = CInt(s1)
        b = CInt(s2)
        c = a - b
        MsgBox("Subtraction=" + c.ToString)
    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click
        a = CInt(s1)
        b = CInt(s2)
        c = a * b
        MsgBox("Multiplication=" + c.ToString)
    End Sub

    Private Sub Button4_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button4.Click
        a = CInt(s1)
        b = CInt(s2)
        c = a / b
        MsgBox("Division=" + c.ToString)
    End Sub
End Class

File: CS1/console/WindowsFormsApplication4/WindowsFormsApplication4/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using System.Data.SqlClient;


namespace WindowsFormsApplication4
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {

            string name =textBox1.Text;
            int rno=Convert.ToInt32(textBox2.Text);
            int marks =Convert.ToInt32(textBox3.Text);


            SqlConnection conn = new SqlConnection("Data Source=MARUTHI;Initial Catalog=sam;Integrated Security=True");

            SqlCommand comm = new SqlCommand("insert into student values(" + "'" + name + "'" + "," + rno + "," + marks + ")",conn);

            conn.Open();

            int i=comm.ExecuteNonQuery();

            if (i == 1)
            {

                MessageBox.Show("Data inserted");
            }

            conn.Close();












        }

        private void button2_Click(object sender, EventArgs e)
        {


           // string name = textBox1.Text;
            int rno = Convert.ToInt32(textBox2.Text);
           int marks = Convert.ToInt32(textBox3.Text);


            SqlConnection conn = new SqlConnection("Data Source=MARUTHI;Initial Catalog=sam;Integrated Security=True");

            SqlCommand comm = new SqlCommand("update  student set marks="+marks+"where rollno="+rno, conn);

            conn.Open();

            int i = comm.ExecuteNonQuery();

            if (i == 1)
            {

                MessageBox.Show("Data Updated");
            }

            conn.Close();






        }

        private void button3_Click(object sender, EventArgs e)
        {

            // string name = textBox1.Text;
            int rno = Convert.ToInt32(textBox2.Text);
           // int marks = Convert.ToInt32(textBox3.Text);


            SqlConnection conn = new SqlConnection("Data Source=MARUTHI;Initial Catalog=sam;Integrated Security=True");

            SqlCommand comm = new SqlCommand("Delete  student  where rollno=" + rno, conn);

            conn.Open();

            int i = comm.ExecuteNonQuery();

            if (i == 1)
            {

                MessageBox.Show("Data Deleted");
            }

            conn.Close();



        }

        private void button5_Click(object sender, EventArgs e)
        {
            textBox1.Text = "";
            textBox2.Text = "";
            textBox3.Text = "";

        }

        private void button4_Click(object sender, EventArgs e)
        {



            // string name = textBox1.Text;
            int rno = Convert.ToInt32(textBox2.Text);
            // int marks = Convert.ToInt32(textBox3.Text);


            SqlConnection conn = new SqlConnection("Data Source=MARUTHI;Initial Catalog=sam;Integrated Security=True");

            SqlCommand comm = new SqlCommand("Select * from  student  where rollno=" + rno, conn);

            conn.Open();




            SqlDataReader dr = comm.ExecuteReader();


            if (dr.Read())
            {
                textBox1.Text = dr[0].ToString();
                textBox3.Text = dr["marks"].ToString();

            }

           

            conn.Close();
        }
    }
}
